app/src/components/OrderBookSnapshot.tsx

```diff
@@ -51,6 +52,18 @@ interface State {
 }
 
 class OrderBookSnapshot extends Component<WithStyles, State> {
+    /**
+     * @desc Handles window resizing and requests a new number of data points appropriate for the new window width
+     */
+    handleResize = debounce(() => {
+        const { selectedDateNano } = this.state;
+        if (selectedDateNano.neq(0)) {
+            const startTime = selectedDateNano.plus(NANOSECONDS_IN_NINE_AND_A_HALF_HOURS);
+            const endTime = selectedDateNano.plus(NANOSECONDS_IN_SIXTEEN_HOURS);
+            this.updateGraphData(startTime, endTime);
+        }
+    }, 100);
```

Comment:
is there a reason we wait a certain amount of time before sending the request again

Reply:
While dragging the browser window to resize it/while dragging the dev tools side bar, many requests are sent to the backend. I found 0.1 seconds to be a reasonable waiting time to minimize the number of requests being sent.